### ISSUE TITLE
Tank improvements

### ISSUE THREAD

**Issue by nickross4444** [SNL-WaterPower/siweed#93]
Tank issues we'd like to address with Kevin:

- [ ] Shrouding IR sensor, currently done with a straw.
- [ ] Siphon or valve to empty tank
- [ ] Metal hardware rusting, maybe new beach altogether.
- [ ] Decal is not visible enough

**Comment by ryancoe** [comment]
@nickross4444 @DeepFriedDerp - please mark if any of these are complete?

**Comment by nickross4444** [comment]
Right only the siphon is done, and the IR shroud and rusting hardware are relatively high priority relative to the rest. @ryancoe ,is Kevin the right person to talk to for those two?

**Comment by ryancoe** [comment]
Yes, Kevin is our guy for this. @DeepFriedDerp - Can you reach out and CC me?
